test/setTimerTest.py

```diff
@@ -0,0 +1,56 @@
+from appium.webdriver.common.appiumby import AppiumBy
+from selenium.webdriver.support.expected_conditions import presence_of_element_located
+from selenium.webdriver.support.wait import WebDriverWait
+
+from base.DriverClass import Driver
+from base.BasePage import BasePage
+
+
+def test_count_down():
+    driver1 = Driver()
+    driver = driver1.getDriverMethod()
+
+    bp = BasePage(driver)
+    element = bp.waitForElement("android:id/button1","id")
+    element.click()
+
+    allow_notification = bp.waitForElement("com.android.permissioncontroller:id/permission_allow_button", "id")
+    allow_notification.click()
+
+    for i in range(3):
+        ok_got_it_btn = bp.waitForElement("com.sportstracklive.stopwatch:id/dismiss_button", "id")
+        ok_got_it_btn.click()
+
+    count_down = bp.waitForElement("com.sportstracklive.stopwatch:id/countDown", "id")
+    count_down.click()
+
+    frame_layout = bp.waitForElement('new UiSelector().className("android.widget.FrameLayout").instance(0)',"uiautomator")
+    if frame_layout.is_displayed():
+        print("\nFrameLayout found and visible.")
+    else:
+        print("\nFrameLayout not found.")
+
+    driver.swipe(780, 1000, 780, 500, 800)
```

Comment:
Here you need to swipe specific amount of time. Add logic to swipe according desired time.